std/algorithm/sorting.d

```diff
@@ -2626,6 +2626,120 @@ unittest
     }
 }
 
+// Always leaves median in r[2]. Credit goes to Xinok, see e.g.
+// http://stackoverflow.com/questions/11065963/possible-to-partition-five-elements-by-median-with-six-comparisons
+private void medianOf5(alias less, Range)(Range r)
+{
+    assert(r.length == 5);
```

Comment:
This failing always indicates a bug inside std.algorithm so there's no need to add 5 lines of scaffolding for it.